src/main/java/jdbchomework/dao/hibernate/AbstractHibDao.java

```diff
@@ -28,16 +27,9 @@ public AbstractHibDao(String entityName, SessionFactory sessionFactory, Class<T>
 
     @Override
     public void add(T toAdd) {
-        Transaction transaction = null;
-
         try (Session session = sessionFactory.openSession()) {
-            transaction = session.beginTransaction();
             session.save(toAdd);
-            transaction.commit();
         } catch (HibernateException e) {
-            if (transaction != null) {
-                transaction.rollback();
-            }
             log.error(CANNOT_CONNECT_TO_DB, e);
             throw new ProblemDbConnection(CANNOT_CONNECT_TO_DB, e);
```

Comment:
Is this class responsible for Exception or some kind of Connection? Please clarify the name.